src/Domain/Tags.hpp

```diff
@@ -496,14 +496,17 @@ struct InterfaceSubitemsImpl {
     }
   }
 
+  using return_type = void;
```

Comment:
Ah, it's only used to signal to the DataBox that it's return-by-reference. I'll add a comment because that certainly is not at all clear. The DataBox internally figures out what the actual type is based of the subtags.